tests/functional/syntax/test_address_code.py

```diff
@@ -125,7 +125,7 @@ def test_address_code_compile_error(
 ):
     with pytest.raises(error_type) as excinfo:
         compiler.compile_code(bad_code)
-    assert type(excinfo.value) == error_type
+    assert excinfo.value is error_type
```

Comment:
actually doesn't `pytest.raises` already check for this?

Reply:
Damn this test was broken I just fixed in https://github.com/vyperlang/vyper/pull/3704/commits/06194773298d59c650ee0ef2dd8413f61bdad89a
But this line is indeed not needed at all